llvm/test/CodeGen/AArch64/aarch64-isel-umin.ll

```diff
@@ -0,0 +1,82 @@
+; RUN: llc -mtriple=aarch64-- -o - < %s | FileCheck %s --check-prefixes=CHECK,CHECK-SD
+; RUN: llc -mtriple=aarch64-- -mattr=+cssc -o - < %s | FileCheck %s --check-prefixes=CHECK,CHECK-CSSC
```

Comment:
Remove the common check, since these are never expected to match. That should remove the "These prefixes are unused" note at the bottom. 
```suggestion
; RUN: llc -mtriple=aarch64-- -o - < %s | FileCheck %s --check-prefix=CHECK
; RUN: llc -mtriple=aarch64-- -mattr=+cssc -o - < %s | FileCheck %s --check-prefix=CHECK-CSSC
```